chromium_src/chrome/browser/history/history_tab_helper.cc

```diff
@@ -0,0 +1,45 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/request_otr/common/buildflags/buildflags.h"
+#include "build/build_config.h"
+
+#if BUILDFLAG(ENABLE_REQUEST_OTR)
+
+#include "chrome/browser/history/history_tab_helper.h"
+
+#include "brave/components/request_otr/browser/request_otr_tab_storage.h"
+#include "chrome/browser/profiles/profile.h"
```

Comment:
please add a comment explaining that these are here to avoid overriding `IsOffTheRecord` in them

Reply:
done